django/db/models/base.py

```diff
@@ -261,7 +262,10 @@ def __new__(cls, name, bases, attrs):
                         'with field of similar name from '
                         'abstract base class %r' % (field.name, name, base.__name__)
                     )
-                new_class.add_to_class(field.name, copy.deepcopy(field))
```

Comment:
I don't get why virtual fields have special handling, why can't we have `parent_fields = base._meta.local_fields + base._meta.local_many_to_many + base._meta.local_virtual_fields` (note that `local_virtual_fields` doesn't exist yet).